open-machine-learning-jupyter-book/data-science/working-with-data/working-with-data.ipynb

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
<p>When I view it locally, the image is still local. Please check if it has been changed to a cos link. If so you can ignore this comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>